content/sensu-go/6.1/observability-pipeline/observe-process/handlers.md

```diff
@@ -44,11 +45,82 @@ TCP and UDP handlers enable Sensu to forward event data to arbitrary TCP or UDP
 
 ## Handler sets
 
-Handler set definitions allow you to use a single named handler set to refer to groups of handlers (individual collections of actions to take on event data).
+Handler set definitions allow you to use a single named handler set to refer to groups of handlers.
+The handler set becomes a collection of individual actions to take (via each included handler) on event data.
+
+For example, suppose you have already created these two handlers:
+
+- `elasticsearch` to send all observation data to Elasticsearch.
+- `opsgenie` to send non-OK status alerts to your OpsGenie notification channel.
+
+You can list both of these handlers in a handler set to automate and streamline your workflow, specifying `type: set`:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+type: Handler
+api_version: core/v2
+metadata:
+  name: send_events_notify_operator
+  namespace: default
+spec:
+  handlers:
+  - elasticsearch
+  - opsgenie
+  type: set
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "Handler",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "send_events_notify_operator",
+    "namespace": "default"
+  },
+  "spec": {
+    "type": "set",
+    "handlers": [
+      "elasticsearch",
+      "opsgenie"
+    ]
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+Now you can route observation data to Elasticsearch and alerts to OpsGenie with a single handler definition, the `send_events_notify_operator` handler set.
 
 {{% notice note %}}
-**NOTE**: Attributes defined on handler sets do not apply to the handlers they include.
+**NOTE**: Attributes defined in handler sets do not apply to the handlers they include.
 For example, `filters` and `mutator` attributes defined in a handler set will have no effect on handlers.
+Define these attributes in individual handlers instead.
+{{% /notice %}}
+
+## Handler stacks
+
+Handler stacks are a group of handlers or a handler set that escalate events through a series of different handlers.
+For example, suppose you want a handler stack with three levels of escalation:
+
+- Level 1: On the first occurrence, attempt remediation.
+- Level 2: On the fifth occurrence, send an alert to Slack.
+- Level 3: On the tenth occurrence, send an alert to PagerDuty.
+Continue to send this alert on every tenth occurrence thereafter until the incident is resolved.
+
+A handler stack for this scenario requires three handlers to take the desired actions based on three corresponding event filters that control the escalation levels:
+
+- Level 1 requires an event filter with the built-in [`is_incident` filter][30] and a corresponding remediation handler.
```

Comment:
This will also require an `occurrences == 1 filter` if **NOT** using a remediation handler that does its own occurrences checking.